docs/articles/documentation/reference/test-api/testcontroller/setfilestoupload.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: docs
+title: t.setFilesToUpload Method
+permalink: /documentation/reference/test-api/testcontroller/setfilestoupload.html
+---
+# t.setFilesToUpload Method
```

Comment:
# Cross-links

API reference topics should also link each other if they are related closely.

For instance, this topic, `t.setFilesToUpload`, should have a link to `t.clearUpload`.

You can say somewhere near the end: `The [t.clearUpload](link) method allows you to remove files added with t.setFilesToUpload.`

And visa versa. 

Similarly, `t.switchToIframe` should link to `t.switchToMainWindow` and visa versa.

`t.resizeWindow` to `t.resizeToFitDevice` - like `You can also use the ... method to resize [in a different manner] ...`

Select actions I suppose should also be cross-linked.

Not sure about clicks - they emulate different user actions after all.

Reply:
* vice versa :)

Will cross-link the relevant ones, thanks!